src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2190,4 +2202,15 @@ PeerImp::getScore (bool haveItem)
    return score;
 }
 
+bool
+PeerImp::highLatency ()
```

Comment:
This class has no coherent notion of constness. Given that, trying to implement it on a function that's logically const but internally does change the state of the object (though it puts it back before it's done) doesn't seem worth any effort.